src/components/modal/z-modal/index.tsx

```diff
@@ -39,6 +39,10 @@ export class ZModal {
   @Prop()
   closable?: boolean = true;
 
+  /** if true, the modal content is scrollable */
+  @Prop()
+  scrollable?: boolean = true;
```

Comment:
Se inizializzi la prop con un default entra in gioco l'inferenza di tipo, quindi boolean diventa superfluo

Reply:
Maledetto linter!